backend/src/test/java/com/woowacourse/naepyeon/service/AuthServiceTest.java

```diff
@@ -58,8 +68,24 @@ void createToken() {
     }
 
     @Test
-    @DisplayName("사용자 정보가 DB에 없는 경우 DB에 사용자 정보를 추가하고 토큰을 반환한다.")
-    void createTokenWithNotExistMember() {
+    @DisplayName("사용자 정보를 받아 네이버 Oauth로 토큰을 만들어 반환한다.")
+    void createNaverToken() {
+        final PlatformUserDto platformUserDto = new PlatformUserDto("seungpang", "email@email.com", "NAVER", "2");
+        when(naverPlatformUserProvider.getPlatformUser(anyString(), anyString(), anyString())).thenReturn(platformUserDto);
```

Comment:
라인이 넘어가네요. 개인적으론 `anyString()`이 아닌 `any()`로 넣어도 괜찮을 것 같은데 승팡 생각은 어떤가요?
```suggestion
        when(naverPlatformUserProvider.getPlatformUser(any(), any(), any())).thenReturn(platformUserDto);
```